mlflow/transformers/__init__.py

```diff
@@ -1188,6 +1189,8 @@ def persist_pretrained_model(model_uri: str) -> None:
     """
     # Check if the model weight already exists in the model artifact before downloading
     root_uri, artifact_path = _get_root_uri_and_artifact_path(model_uri)
+    if ModelsArtifactRepository._is_logged_model_uri(root_uri):
+        root_uri = ModelsArtifactRepository.get_underlying_uri(root_uri)
```

Comment:
Can we update `ModelsArtifactRepository.log_artifacts` to support logging if the URI is `models:/<model_id>`? Does that work?

Reply:
Yes it should work as well!

Reply:
But we still need to forbidden log_artifaccts for `models:/<name>/<version>` case